.github/workflows/codeql_analysis.yml

```diff
@@ -7,9 +7,14 @@ name: 'CodeQL'
 
 on:
   push:
+    branches:
+      - master
+      - next
   pull_request:
-  schedule:
```

Comment:
No need to run on a scheduled since we will be running on each PR and push event.